eng/packages/http-client-csharp-mgmt/emitter/src/index.ts

```diff
@@ -2,3 +2,4 @@
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
 export { $lib } from "./lib/lib.js";
+export { $onEmit } from "./emitter.js";
```

Comment:
we need this for compiler to discover this emitter

Reply:
Interesting - how was it working before?

Reply:
mgmt CI did not run in PRs, it has been failing for a while